resolwe/flow/models/process.py

```diff
@@ -180,6 +180,10 @@ def get_resource_limits(self):
 
         limits['cores'] = int(resources.get('cores', 1))
 
+        max_cores = getattr(settings, 'FLOW_MAX_CORES_PER_PROCESS', None)
```

Comment:
I would suggest `FLOW_PROCESS_MAX_CORES` since it would be similar to `FLOW_PROCESS_RESOURCE_DEFAULTS` and `FLOW_PROCESS_RESOURCE_OVERRIDES`.